toml/default_parameters.toml

```diff
@@ -466,3 +454,34 @@ value = 7.85
 type = "float"
 description = "Intercept of line relating leaf-level fluorescence to spectrometer-observed fluorescence as a function of Vcmax 25 (leaf level). Lee et al 2015."
 tag = "SIFParameters"
+
+# Canopy roughness
+["canopy_z_0min"]
+value = 0.013
+type = "float"
+description = "Minimum roughness"
+tag = "ConstantRoughnessModel"
```

Comment:
I don't see `ConstantRoughnessModel` defined in the src code, should this be `MoninObukhovCanopyFluxes`?